All_materials/20230822_Web_Scraping_Financial_Ombudsman_Service/config.toml

```diff
@@ -0,0 +1,18 @@
+project_name = "fos_web"
+
+[web]
+seach_term = "scam"
+start_date = "2023-06-01"
+end_date = "2023-06-02"
+
+[table]
+ref = 'Ref'
+date = 'Date'
+bank = 'Bank'
+scale_of_lost = 'Scale of Lost'
+outcome = 'Outcome'
+CRM = 'CRM'
+summary = 'Sample'
+
+[file]
```

Comment:
could this be more informatively named 'output_file'?

Reply:
Changed to table_of_complaints